TelegramBot/BotHandler.cs

```diff
@@ -40,10 +41,10 @@ public void OnMessageHandler(object sender, MessageEventArgs e)
                 return;
             var userRequest = parser.ParseMessage(e.Message.Text);
             userRequest.User = new TelegramUser(e.Message.From.Id);
-            userRequest.telegramMessage = e.Message;
+            userRequest.TelegramMessage = e.Message;
             if (userRequest.Command == null)
             {
-                var commandType = commandTypeService.GetCurrentCommandType(userRequest.User);
+                commandTypeService.TryGetCurrentCommandType(userRequest.User, out var commandType);
                 if (commandType.Name == "NoCommand")
```

Comment:
Теперь можно так сделать

```suggestion
                if (commandTypeService.TryGetCurrentCommandType(userRequest.User, out var commandType))
```

Reply:
Раз `NoCommand` больше нет